cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -29,17 +29,23 @@ import com.beust.jcommander.Parameters
 
 import com.here.ort.CommandWithHelp
 import com.here.ort.model.OrtResult
+import com.here.ort.model.config.Resolutions
 import com.here.ort.model.readValue
+import com.here.ort.reporter.DefaultResolutionProvider
 import com.here.ort.reporter.Reporter
+import com.here.ort.reporter.ResolutionProvider
 import com.here.ort.utils.PARAMETER_ORDER_MANDATORY
+import com.here.ort.utils.PARAMETER_ORDER_OPTIONAL
 import com.here.ort.utils.log
 import com.here.ort.utils.safeMkdirs
 import com.here.ort.utils.showStackTrace
 
 import java.io.File
 
-@Parameters(commandNames = ["report"],
-        commandDescription = "Present Analyzer and Scanner results in various formats.")
+@Parameters(
+    commandNames = ["report"],
+    commandDescription = "Present Analyzer and Scanner results in various formats."
+)
```

Comment:
Is this reformatting (and below) intended?

Reply:
No, undone.